app/helpers/money_helper.rb

```diff
@@ -0,0 +1,17 @@
+module MoneyHelper
```

Comment:
Does it make sense to put this in helpers? I think we're using helpers as Rails helpers, which are view things. Is this just for frontend, or are we going to use it in other places?

Reply:
I think so since it's meant to abstract away from calls to `money_without_cents_and_with_symbol`, which is a view helper method, and it accesses the localized currency string. `money_usd` could be private, but I'm not sure there's straightforward to do in a view helper module.